tests/gc/gcnone.nim

```diff
@@ -0,0 +1,6 @@
+discard """
+  matrix: "--gc:none -d:useMalloc"
```

Comment:
Since testament specializes the "gc" directory, this test is never run. --> We need to tweak testament for category "gc" in order to prevent this from happening the next time.